.github/workflows/try-runtime.yml

```diff
@@ -0,0 +1,64 @@
+name: Try-runtime
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
```

Comment:
Does `edited` get triggered on commits? Otherwise I don't think we need to re-run this when changing title or description.

Reply:
Removed